main.go

```diff
@@ -494,6 +494,7 @@ func _main(ctx context.Context) error {
 
 	// Create the explorer system.
 	explore := explorer.New(&explorer.ExplorerConfig{
+		Client:          dcrdClient,
```

Comment:
I think this is a clue that `TxAtomicSwapsInfo` function should be routed through a method on the `ChainDB`, which already has a copy of the `rpcclient.Client` and the `chaincfg.Params`. It also fits the prevailing model of `Explorer` and the `explorerDataSource` interface.

Reply:
`TxAtomicSwapsInfo` was a method on the `ChainDB`, but since the method doesn't _really_ perform a db query, @chappjc and I felt it was better to make it a `rpcutils` function, as the method _really_ only need to make rpc calls. Related conversation: https://github.com/decred/dcrdata/pull/1733#discussion_r411545218 and https://github.com/decred/dcrdata/pull/1733#discussion_r411548496.

I italicised _really_ because there is one area where a db call is necessary, when calling `TxAtomicSwapsInfo` from the api route handler. The db call is not necessary when accessing the tx explorer page, so explorer really does not need to make db calls from the `TxAtomicSwapsInfo` method.

Re the current `Explorer` / `explorerDataSource` model, we've discussed making some changes there in [chat](https://matrix.to/#/!LmKzrmxJIXNHNiEmIh:decred.org/$158740395316769DahOV:decred.org?via=decred.org&via=matrix.org&via=decred.org), to not mix rpc calls and db calls in one db pkg.